### PR TITLE
Add inner shadow to top bar on group-related pages

### PR DESCRIPTION
**Pull request by broskoski**
screen shot 2018-12-12 at 3 01 00 pm
screen shot 2018-12-12 at 3 01 10 pm
screen shot 2018-12-12 at 3 01 34 pm

### CODE REVIEW

File: apps/channel/queries/channelCan.js [hunk 1]
```diff
@@ -4,6 +4,15 @@ export default gql`
...
       id
+      owner {
+        __typename
+        ... on Group {
+          id
+        }
+        ... on User {
+          id
+        }
+      }
```
Comment (dzucconi) [inline apps/channel/queries/channelCan.js]: So this isn't related to channelCan and shouldn't be in here. Could just make this it's own fragment.

Comment (dzucconi) [reply apps/channel/queries/channelCan.js]: OR—just rename this since it's legacy

Comment (dzucconi) [reply apps/channel/queries/channelCan.js]: legacyChannelData or somethin

Comment (broskoski) [reply apps/channel/queries/channelCan.js]: I shall

**Pull request merged**